README.md

```diff
@@ -62,7 +62,7 @@ OpenEdge projects can be configured in a file called `openedge-project.json`. Th
     {
       "name": "sp2k", // Logical name of database, or physical name if -ld is not used
       "connect": "-db db/sp2k -RO", // ABL connection string
-      "schemaFile": "dump/sp2k.df",   // Database schema, DF format
+      "schemaFile": "dump/sp2k.df",   // Database schema, DF format. This can be dumped with Data Administration:  https://docs.progress.com/bundle/openedge-database-tools-122/page/Dump-data-definitions.html
```

Comment:
There is the `PCTDumpSchema` ant task and the `prodict/dump_df.p` openedge procedure that can be used in lieu of Data Admin.  I'd also question why the schema isn't part of the repo already.

Is mentioning Data Admin here appropriate when there are other options, or is that too prescriptive?